recipes/r-micropan/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = '1.1.2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-micropan
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: micropan_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/micropan_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/micropan/micropan_{{ version }}.tar.gz
+
+  sha256: efb4491b04f781a99fb9180689b7788618800618c3433a8af897ee926f556a67
+
+build:
+  number: 0
+  skip: true  # [win32]
```

Comment:
You would need to skip Windows because r-igraph is skipped on Windows:

https://github.com/conda-forge/r-igraph-feedstock/blob/master/recipe/meta.yaml#L20